ci/pipelines/shared/jinja.variables.yml

```diff
@@ -67,6 +67,7 @@ java_test_versions:
 
 metadata:
   initial_version: 1.14.0-((semver-prerelease-token)).0
+  mass_test_run_iterations: 200
```

Comment:
@mhansonp I know there's been discussion in the past about how much value we get out of 100 vs 200.  Can you mention the benefits of 200 vs 100?  My back-of-the-envelope calculations suggest 200 takes about 4 days, while 100 could be finished in a weekend (and about $900 cheaper, if anyone cares)

Reply:
Basically, 200 gives us a big enough sample size to tell the 1 off from the 2 or more. I think running 100 might be fine. Especially if the code is doing a last 200 check.

Reply:
Why would I run 100 jobs of `abcdef` and then run a report containing 200 jobs mixed of `abcdef` and `123456` ?

Reply:
So you would use the SHA as normal, but you would run 100 tests and report against the last 200. That will give us a historical result set for the last two weeks. Or to just keep it simple, we can just do 100 and report on 100.